test/integration/node_pool/controls/kubectl.rb

```diff
@@ -79,11 +79,6 @@
               key: "all-pools-example",
               value: "true",
             },
-            {
```

Comment:
Why was this removed?

Reply:
For some reason `node-pool` test started to fail when checking `taints`. See `run-test-node-pool` in https://concourse.infra.cft.tips/teams/cft/pipelines/terraform-google-kubernetes-engine/jobs/integration-tests/builds/453 and https://concourse.infra.cft.tips/teams/cft/pipelines/terraform-google-kubernetes-engine/jobs/integration-tests/builds/452.

After making the change in `kubectl.rb` the test started passing again. See: https://concourse.infra.cft.tips/teams/cft/pipelines/terraform-google-kubernetes-engine/jobs/integration-tests/builds/454

I will revert the change to this test case along with `location` changes and i'll see if the test will still fail.

Reply:
If this fixes CI, it should be a separate PR regardless

Reply:
I'll revert the change first and see what happens. If master branch is passing and my branch is failing than i don't see how separate PR would fix this.

Reply:
If that's the case, we should probably understand why this metadata change is integral to tests passing. Let's see how it goes with the reversion.

Reply:
Change reverted. PR https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/171 opened to fix this issue.

Reply:
Thanks!